urbanopt/geojson-modelica-translator
# Sys param rearranging

#### Any background context you want to provide?
This is followup work to #273 
#### What does this PR accomplish?
Moves the `ets_model_parameters` block of a sys-param file inside the building block, so there is an ets for each building.
#### How should this be manually tested?
`py.test`

However, `py.test tests/system_parameters/test_system_parameters.py::SystemParametersTest::test_get_param_with_none_building_id` fails with:

`AssertionError: {'indirect': {'heat_flow_nominal': 8000, [534 chars] 20}} != 'Not None'`

I suspect the failure is due to [this intentional behavior](https://github.com/urbanopt/geojson-modelica-translator/blob/sys-param-rearranging/geojson_modelica_translator/system_parameters/system_parameters.py#L188-L192). Potential ways of handling this:

1. Don't add the ets block to that portion of the test sys-param file (revert 2306fd3)
2. Change the test to expect different results
3. Change the actual code to not return defaults if missing a building_id

#### What are the relevant tickets?
#279 
#### Screenshots (if appropriate)
